java/webflux-client/src/main/java/no/bankaxept/epayment/client/webflux/WebFluxClient.java

```diff
@@ -18,11 +19,17 @@ public class WebFluxClient implements HttpClient {
   private final WebClient webClient;
 
   public WebFluxClient(String baseUrl) {
-    webClient = WebClient.create(baseUrl);
+    webClient = Optional.ofNullable(WebFluxClient.class.getPackage().getImplementationVersion())
```

Comment:
getPackage() can also return null, could we improve null safety? Unless you think it's entirely impossible to run into null there. I'm worried for direct instantiations of WebFluxClient, which we have in main project.

Reply:
We know that the `WebFluxClient.class.getPackage()` never returns null, but `implementationVersion()` might return null if the package is not loaded from the jar file where the implementation version is set in the `META-INF/MANIFEST.MF` file.

Reply:
Discussed on daily 👍